src/main/scala/akka/persistence/cassandra/CassandraPluginConfig.scala

```diff
@@ -65,7 +64,10 @@ class CassandraPluginConfig(config: Config) {
       keyStorePath,
       keyStorePW )
 
-    clusterBuilder.withSSL(new SSLOptions(context,SSLOptions.DEFAULT_SSL_CIPHER_SUITES))
+    // FIXME there is also a NettySSLOptions?  
+    clusterBuilder.withSSL(JdkSSLOptions.builder.withSSLContext(context).build())
+        // FIXME what about? .withCipherSuites(SSLOptions.DEFAULT_SSL_CIPHER_SUITES))  
+//    clusterBuilder.withSSL(new SSLOptions(context, SSLOptions.DEFAULT_SSL_CIPHER_SUITES))
```

Comment:
Does anyone know what SSL options to use here? The old `SSLOptions.DEFAULT_SSL_CIPHER_SUITES` doesn't exist any more.